src/formatter/simple.rs

```diff
@@ -1,361 +1,433 @@
+use std::io;
+use std::mem;
+
+use colored::*;
```

Comment:
I'm not experienced with Rust on Windows, I have to admit. Have a suggestion for what to put there?